spec/controllers/catalog_controller_spec.rb

```diff
@@ -974,6 +974,54 @@
       end
     end
 
+    describe '#fetch_ovf_template_details' do
+      let(:user)                     { FactoryBot.create(:user_with_group) }
+      let(:admin_user)               { FactoryBot.create(:user, :role => "super_administrator") }
+      let(:ems)                      { FactoryBot.create(:ems_vmware) }
+      let(:container_template)       { FactoryBot.create(:container_template, :ext_management_system => ems) }
```

Comment:
I don't think a container template is the kind of template you're looking for here